home/.zsh/functions/docker.zsh

```diff
@@ -16,23 +88,31 @@ function dkr-up {
 }
 
 function dkr-bash {
-    dkr-container-name ${1}
-
-    dkr-exec "${__container}_1" bash
+    dkr-exec "$1" bash
 }
 
 function dkr-zsh {
-    dkr-container-name ${1}
+    if [[ -z $1 ]]; then
```

Comment:
Why doesn't dkr-bash need this same check?

Reply:
Good catch.